src/main/java/hw8/entities/MetalsColorsInput.java

```diff
@@ -0,0 +1,81 @@
+package hw8.entities;
+
+import com.epam.jdi.tools.DataClass;
+import hw8.Utils;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.lang.String.format;
+
+public class MetalsColorsInput extends DataClass<MetalsColorsInput> {
+    private List<String> summary;
+    private List<String> elements;
+    private String color;
+    private String metals;
+    private List<String> vegetables;
+
+    private static Map<String, Integer> summaryValuesIndexes = new HashMap<>();
+
+    static {
+        summaryValuesIndexes.put("1", 1);
+        summaryValuesIndexes.put("2", 1);
+        summaryValuesIndexes.put("3", 2);
+        summaryValuesIndexes.put("4", 2);
+        summaryValuesIndexes.put("5", 3);
+        summaryValuesIndexes.put("6", 3);
+        summaryValuesIndexes.put("7", 4);
+        summaryValuesIndexes.put("8", 4);
+    }
+
+    public MetalsColorsInput(List<String> summary, List<String> elements, String color, String metals, List<String> vegetables) {
+        this.summary = summary;
+        this.elements = elements;
+        this.color = color;
+        this.metals = metals;
+        this.vegetables = vegetables;
+    }
+
+    public SummaryEntity getSummaryEntity() {
```

Comment:
Typical example of overdesign, should be simplified.

Reply:
Summary Entity is deleted